docs/atomic-structure.md

```diff
@@ -259,12 +241,11 @@ An explanation for the usage of each bundle is contained in scripts.js.
 
 Routes are used to serve JavaScript bundles to the browser based
 on the requested URL or Wagtail page Media property.
-This happens via code contained in base.html.
+This happens via code contained in [base.html](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_layouts/base.html#L236-L285).
 
 #### Base.html
 
-This file serves as the base document for serving up assets and content.
-It's currently very complicated, obtrusive, and needs to be refactored.
+This file serves as the base HTML template for serving up assets and content. [View base.html on Github](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_layouts/base.html).
```

Comment:
This sentence could be rolled up into the section above. Having an entire section describing `base.html` seems unnecessary.